### PR TITLE
Fixed the path to OCaml for Windows installation

### PR DESCRIPTION
**Pull request by anitagov**

### CODE REVIEW

File: docs/GettingStartedDocs/GettingStarted.Windows.md [hunk 1]
```diff
@@ -23,7 +23,7 @@ Windows.
 - [Microsoft Visual Studio 2017](https://www.visualstudio.com/downloads/)
 - [Git for Windows 64-bit](https://git-scm.com/download/win)
 - [clang (preferably version 7.0 or above)](http://releases.llvm.org/download.html)  
-- [OCaml on Windows 64-bit](https://fdopen.github.io/opam-repository-mingw/installation/) (Note that this will install a MinGW environment for OCaml.)
+- [OCaml on Windows 64-bit](https://www.ocamlpro.com/pub/ocpwin/ocpwin-builds/ocpwin64/20160113/ocpwin64-20160113-4.02.1+ocp1-mingw64.exe) (Note that this will install a MinGW environment for OCaml.)
```
Comment (andschwa) [inline docs/GettingStartedDocs/GettingStarted.Windows.md]: Hm, I am not sure about hard-coding to a particular version of an installer. I liked just linking to the home page of the installer (like the other links do).

Comment (anakrish) [reply docs/GettingStartedDocs/GettingStarted.Windows.md]: This particular homepage is quite hard to read: https://www.ocamlpro.com/pub/ocpwin/ocpwin-builds/ocpwin64/20160113/
Maybe provide both.

Comment (anitagov) [reply docs/GettingStartedDocs/GettingStarted.Windows.md]: OK. Will incorporate your inputs. Will also recommend that the .exe installer be used.

**Pull request merged**